### ISSUE TITLE
Inventory select method doc problem

### ISSUE THREAD

**Issue by bonaime** [obspy/obspy#1210]
Part of the documentation of the select method of "obspy.core.inventory.inventory.Inventory" is missing:
"select Returns the Inventory object with only the"

http://docs.obspy.org/master/packages/autogen/obspy.core.inventory.inventory.Inventory.html#obspy.core.inventory.inventory.Inventory
and
http://docs.obspy.org/packages/autogen/obspy.station.inventory.Inventory.html#obspy.station.inventory.Inventory

**Comment by megies** [comment]
Uhm, well.. the box in the overview just can't fit the whole text, I guess? It's just taking whatever comes first in the full docstring:
http://docs.obspy.org/packages/autogen/obspy.station.inventory.Inventory.select.html#obspy.station.inventory.Inventory.select

**Comment by bonaime** [comment]
There is something strange 
![capture d ecran 2017-05-09 a 09-05-17 14 16 26](https://cloud.githubusercontent.com/assets/750709/25850413/70ec29a8-34c2-11e7-88a8-a9dda0303ca7.png)

for estimate_magnitude, the first sentence is completely visible in 
https://docs.obspy.org/packages/autogen/obspy.signal.invsim.html#module-obspy.signal.invsim
```code
def estimate_magnitude(paz, amplitude, timespan, h_dist):
    """
    Estimates local magnitude from poles and zeros of given instrument, the
    peak to peak amplitude and the time span from peak to peak.
```

for estimate_wood_anderson_amplitude, only the first line of the method docstring is visible !

```code
def estimate_wood_anderson_amplitude(paz, amplitude, timespan):
    """
    Convert amplitude in counts measured of instrument with given Poles and
    Zeros information for use in :func:`estimate_magnitude`.
```

**Comment by megies** [comment]
Not sure we can do much about it.. that's sphinx internals I guess. I assume that sphinx is showing everything if it is at most X characters, otherwise just the first line. Dunno. Not much we can do.

**Comment by megies** [comment]
Fixed by #2236
